include/dt-bindings/clock/mcux_lpc_syscon_clock.h

```diff
@@ -15,12 +15,19 @@
 #define MCUX_FLEXCOMM5_CLK		5
 #define MCUX_FLEXCOMM6_CLK		6
 #define MCUX_FLEXCOMM7_CLK		7
-#define MCUX_PMIC_I2C_CLK		16
-#define MCUX_HS_SPI_CLK		8
-#define MCUX_USDHC1_CLK		9
-#define MCUX_USDHC2_CLK		10
+#define MCUX_FLEXCOMM8_CLK		8
+#define MCUX_FLEXCOMM9_CLK		9
+#define MCUX_FLEXCOMM10_CLK		10
+#define MCUX_FLEXCOMM11_CLK		11
+#define MCUX_FLEXCOMM12_CLK		12
+#define MCUX_FLEXCOMM13_CLK		13
+#define MCUX_HS_SPI_CLK		14
+#define MCUX_PMIC_I2C_CLK		15
 
-#define MCUX_CTIMER_CLK_OFFSET  11
+#define MCUX_USDHC1_CLK		20
+#define MCUX_USDHC2_CLK		21
+
+#define MCUX_CTIMER_CLK_OFFSET  22
```

Comment:
nit: mixes tabs/spaces, please align properly

Reply:
Thanks for pointing this, fixed.